virttest/libvirt_xml/devices/disk.py

```diff
@@ -240,6 +240,30 @@ def new_drivermetadata(self, **dargs):
             setattr(new_one, key, value)
         return new_one
 
+    def get_all_backingstore(self):
+        """
+        Get all backingstore of a Disk object
+
+        :return: an ordered list of backingstore items
+        """
+
+        def _get_next_backingstore(elem):
+            """
+            Recursively get backingstore object
+
+            :param elem: root element to get backingstore object
+            """
+            if elem.xmltreefile.find('/backingStore') is None:
```

Comment:
May I ask whether find method here comes from ElementTree ?

Reply:
@chunfuwen Sorry that I just assumed we have already known about this method `xmltreefile.find()` because we use it constantly. But seems you're not familiar with it, and that's ok, I'm happy to explain to you. 

You could find the definition here: https://github.com/avocado-framework/avocado-vt/blob/master/virttest/element_tree.py 

If you need any help with the code, please don't hesitate to contact me. I'm always happy to help. :handshake:

Reply:
@chloerh 
You insists on 3038 is better than https://github.com/avocado-framework/avocado-vt/pull/3062 since 3062 use primitive methods from elementTree.
So my question is why you still need use primitive method : find from elementTree?

Reply:
@chunfuwen 

First of all, I never `insists on 3038 is better than #3062` **since** `3062 use primitive methods from elementTree.` , on the contrary, you're the one who claimed we should choose your way which is more **primitive** (https://github.com/avocado-framework/avocado-vt/pull/3038#discussion_r631711663)

Secondly, I think you might missed the last comment I replied (https://github.com/avocado-framework/avocado-vt/pull/3038#discussion_r636584965), which makes your question hard to understand. To be honest, I don't get your question. 
Maybe you confuse https://github.com/avocado-framework/avocado-vt/blob/master/virttest/element_tree.py  with the primitive elementtree that you imported in #3062 ? I hope this helps.